lib/l10n/app_de.arb

```
@@ -287,8 +287,8 @@
   "deleteAllDataWarning": "Dies wird alle deine Chats, Nachrichten und Einstellungen von diesem Gerät dauerhaft löschen. Diese Aktion kann nicht rückgängig gemacht werden.",
   "deleteAllDataError": "Fehler beim Löschen aller Daten. Bitte versuche es erneut.",
   "chatInformation": "Chat-Informationen",
-  "addAsContact": "Als Kontakt hinzufügen",
-  "removeAsContact": "Als Kontakt entfernen",
+  "follow": "Folgen",
```

RuboCop (1.86.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)